R/utils.R

```diff
@@ -139,3 +139,17 @@ load_raster <- function(urlpaths, bbox = NULL) {
     rasters[[1]]
   }
 }
+
+#' Combine river centerline and surface
+#'
+#' @param river_centerline River line as sfc_LINESTRING or sfc_MULTILINESTRING
+#' @param river_surface River surface as sfc_POLYGON or sfc_MULTIPOLYGON
+#'
+#' @return Combined river as sfc_LINESTRING
```

Comment:
Isn't this a multilinestring instead?
```suggestion
#' @return Combined river as sfc_MULTILINESTRING
```

Reply:
It is, indeed!